ecosystem/sep-0024.md

```diff
@@ -697,6 +695,18 @@ For example:
 }
 ```
 
+## Changes from SEP-6
+
+There is a small set of changes when upgrading from SEP-6 to SEP-24.
+
+1. SEP-24 now requires authentication on all endpoints.  The `authentication_required` flag is removed from the `/info` endpoint since authentication is assumed.
+1. `GET /deposit` and `GET /withdraw` have been replaced with [`POST /transactions/deposit/interactive`](#deposit) and [`POST /transactions/withdraw/interactive`](#withdraw).  This is for security purposes to keep submitted personally identifiable information out of URL query parameters.
```

Comment:
Probably worth pointing out that the endpoints have to be sent multipart/form-data.